drivers/gpu/drm/i915/i915_debugfs.c

```diff
@@ -833,7 +833,33 @@ static const struct file_operations i915_forcewake_fops = {
 	.open = i915_forcewake_open,
 	.release = i915_forcewake_release,
 };
+#ifdef CONFIG_QNX_GUEST
+static int
+i915_force_provisioning_vfs_set(void *data, u64 val)
+{
+	struct drm_i915_private *i915 = data;
+	/* Enable vf by setting 'i915_force_provisioning_vfs' to 1 */
+	if (val > 1)
+		return -EINVAL;
+
+	if (val == 1)
+		(void)i915_sriov_pf_enable_vfs(i915, val);
 
+	return 0;
+}
+
+static int
+i915_force_provisioning_vfs_get(void *data, u64 *val)
```

Comment:
Do we need this function. Everything is hardcoded inside this.

Reply:
If there is a function, user will get 0 from the file. If set the function to NULL, i think it may return an error.
But refer to i915_drop_caches_get as an example,  return a '0'.
    static int
    i915_drop_caches_get(void *data, u64 *val)
    {
	    *val = DROP_ALL;
    
	    return 0;
    }